### ISSUE TITLE
As a user in a large organization, I would like to unsubscribe from build notifications

### ISSUE THREAD

**Issue by afeld** [docker/hub-feedback#686]
I am a member of the [`18fgsa`](https://hub.docker.com/u/18fgsa/) organization on Docker Hub, and while we don't have a _huge_ number of repositories, we have enough that I get automated build failure notifications for repositories I'm not involved with. I would like a way to unsubscribe to individual repository notifications. (Apologies if this option exists and I missed it.) Thanks!

**Comment by bcwalrus** [comment]
Try Settings -> Notifications -> "Notify me when an automated build fails".

![notif](https://cloud.githubusercontent.com/assets/171078/15793687/7e16cad6-299a-11e6-8a11-f5ae012ed542.png)


(Feel free to reopen if that doesn't work for you. Thanks!)

**Comment by afeld** [comment]
Right- that's for notifications from *all* repositories, not on a per repository basis.
